python/environment.yml

```diff
@@ -1,20 +1,23 @@
 name: glow
 channels:
   - bioconda
+  - conda-forge
 dependencies:
-  - python=3.7
   - bedtools
+  - click=7.1.1 # Docs notebook source generation
+  - databricks-cli=0.9.1 # Docs notebook source generation
   - jinja2
-  - numpy=1.17.4
-  - scipy=1.4.1
-  - scikit-learn=0.22.1
   - nomkl # Skip MKL for local development
+  - numpy=1.17.4
   - pandas=0.25.3
   - pip
   - pyarrow=0.15.1
   - pyspark=3.0.0
+  - python=3.7
```

Comment:
I think it's actually standard to list python first since it is pretty important :).